src/styles/components/_create-page__form.css

```diff
@@ -0,0 +1,46 @@
+.create-page__form {
+  max-width: 360px;
+  padding: 18px;
+  margin: 0 auto;
+  display: relative;
+}
+::placeholder {
+  /* Chrome, Firefox, Opera, Safari 10.1+ */
+  color: var(--color--grey);
+  opacity: 1; /* Firefox */
+}
+
+.create-page__form__heading {
+  color: whitesmoke;
+  font-size: 1em;
+  font-weight: bold;
+  margin-block-end: 0.1em;
+  text-shadow: 2px 1px 3px black;
+}
+
+.create-page__form__label {
+  display: block;
+}
+
+.create-page__form__add-answer__counter,
+.create-page__form__add-question__counter,
+.create-page__form__add-tags__counter {
```

Comment:
Please consider a component `counter` that you use three times. Don't recreate the nesting of your html elements as a class name.